subsys/bootloader/bl_crypto/bl_crypto_internal.h

```diff
@@ -41,8 +31,8 @@ bool verify_hash(const u8_t *data, size_t data_len, const u8_t *expected);
  * @param[in]  expected Expected hash
  * @param[in]  hash_len Length of hash
  */
-bool verify_truncated_hash(const u8_t *data, size_t data_len,
-			   const u8_t *expected, size_t hash_len);
+bool verify_truncated_hash(const u8_t *data, u32_t data_len,
```

Comment:
bool external not documented (in all definitions in this file)

Reply:
Will add docs

Reply:
I amended the docs in this file.